web_drag_drop_upload/__openerp__.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright 2014 Agent ERP GmbH
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#
+##############################################################################
+
+{
```

Comment:
Please following template and guidelines here: 
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py
version: 8.0.1.0.0
No description (=>README)
add OCA (Odoo Community Association) in the author list
If Andreas is contributor (individual), remove from author (companies) and add him in the corresponding section of the README